sound/soc/sof/intel/hda.h

```diff
@@ -201,6 +201,7 @@
 #define HDA_ROM_INIT_TIMEOUT		70
 #define HDA_DSP_CTRL_RESET_TIMEOUT		100
 #define HDA_DSP_WAIT_TIMEOUT		500	/* 500 msec */
+#define HDA_DSP_REG_POLL_DELAY		500	/* 0.5 msec */
```

Comment:
@fredoh9 I think we should call it initial delay and maybe add _US to the name to name it clear it is in us

Reply:
I think resolution a more appropriate word than delay but it is too long. so maybe shorten it?

Reply:
sure, will rename it

Reply:
@ranj063 how about this name? HDA_DSP_REG_POLL_INIT_INTERVAL_US.
A bit long but poll usually pairs with interval.

Reply:
@fredoh9 yes interval sounds better. you can even drop init and just adjust the interval within the register_poll() method.

Reply:
sound great. will upload the update shortly.